### ISSUE TITLE
Add searchbar toggle in settings

### ISSUE THREAD

**Issue by smithjl1** [Tautulli/Tautulli#224]
v 1.2.0 added a global search bar; I do not want this search available. Please add an option in settings to disable the global search bar.

**Comment by drzoidberg33** [comment]
It really doesn't take up much space when not being used, I don't think another setting would be the best option here. You could add a Greasemonkey/Tampermonkey script or something to hide it on your install.

**Comment by drzoidberg33** [comment]
I'm marking this as wontfix as I don't think this is something many people would worry too much about and I don't want to unnecessarily clutter the settings menu as it's already quite complicated.

You can use the suggestion I gave above, you can use something like this to hide it:

```
var div = document.getElementById("search_form"); 

if (div) {
    div.style.display = "none";
}
```

**Comment by smithjl1** [comment]
Hrm, apparently my issue was due to rendering. Before, the search bar was
showing below the icon and above the menu bar, which was overlapping the
first 20-30 pixels of the main panel. See attached screenshot:

[image: 10-1-2015 4-51-40 PM.png]
The issue resolved when I reloaded the page, with the search to the far
right now:
[image: pasted1]

While I have the chance, thanks for building such a great app. I have about
20-30 users on my Plex server, and this makes it really easy to monitor and
resolve issues. It's also really easy to install and maintain, something
every other plex monitor fails to do. Big props to you!
